LayoutTests/TestExpectations

```diff
@@ -1464,6 +1464,10 @@ webkit.org/b/133057 fast/table/border-collapsing/collapsed-borders-adjoining-sec
 
 webkit.org/b/169075 editing/selection/extend-by-character-007.html [ Failure ]
 
+# <select> element vertical writing mode support is not yet enabled
+fast/forms/vertical-writing-mode/listbox-vertical-lr-option-order.html [ Failure ]
+fast/forms/vertical-writing-mode/listbox-vertical-rl-option-order.html [ Failure ]
```

Comment:
These new tests could be WPT.

Reply:
See:

https://github.com/web-platform-tests/wpt/blob/master/css/css-writing-modes/forms/select-size-scrolling-and-sizing.optional.html
https://github.com/web-platform-tests/wpt/blob/master/css/css-writing-modes/forms/select-multiple-scrolling.optional.html

Reply:
Rewrote as a WPT.